src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -246,7 +246,7 @@ public async void WorkerError(WorkerErrorEvent workerError)
             if (!_disposing)
             {
                 _logger.LogDebug("Handling WorkerErrorEvent for runtime:{runtime}, workerId:{workerId}", workerError.Language, workerError.WorkerId);
-                _languageWorkerErrors.Add(workerError.Exception);
+                ModifyErrorBucketAndAdd(workerError);   // This either empties the stack and adds the workerError or just adds it to the stack depending on the interval between errors.
```

Comment:
Rename method to: AddOrUpdateErroBucket. Delete comment and method name is self explanatory